Automattic/wp-calypso
# Migration Flow: Add modal suggesting HE help

Related to #

## Proposed Changes

This is an initial task part of the migration flow migration plan pdDR7T-1uw-p2, this should be behind a feature flag and for now, invisible to users.

This PR adds a modal asking users to subscribe to the creator plan and get help from HE on the site migration, this modal should be triggered when the user navigates back using the stepper back button. 

* The modal copy still in progress but not a blocker for this PR
* The “Take the deal” button still without implementation, will be done in a later PR
* This PR does not handle the browser's back button, since we are still not sure how to provide a good experience when navigating back using it.

<img width="1232" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/2c2a73aa-0b92-4d24-b76c-947326dcc691">



## Testing Instructions



* Pull and run this branch or use live link
* Start the migration flow, `/setup/import-hosted-site/import?source=sites-dashboard&ref=hosting-flow`
* Add `flags=migration_assistance_modal` to be able to see the modal when navigating back

## Pre-merge Checklist
